blockchain/src/state/read/model.rs

```diff
@@ -143,6 +146,22 @@ pub trait StateQueries {
         Ok(Some(bytes.into()))
     }
 
+    fn move_bytecode_at(
+        &self,
+        account: AccountAddress,
+        name: &str,
```

Comment:
This code looks good. How can we implement the `listing` function to get all the modules for an account? I'm curious, cause that might affect this method.